zilliqa/src/cfg.rs

```diff
@@ -188,6 +192,20 @@ impl<'de> Deserialize<'de> for Amount {
     }
 }
 
+impl Add for Amount {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
+        Amount::from(*self + *other)
+    }
+}
+
+impl Sub for Amount {
+    type Output = Self;
+    fn sub(self, other: Self) -> Self {
+        Amount::from(*self - *other)
+    }
+}
```

Comment:
Maybe a bit of a nitpick, but I think I'd prefer if we just 'unwrapped' the `Amount` in `state.rs` straight away to get a `u128` and did the computation on that. In my head `Amount` is an awkward workaround because the TOML crate we use doesn't support large integers and it should only exist at the outer-most layer.

I'm not massively opinionated though, so feel free to ignore me if you disagree :)

Reply:
That makes sense. I wasnt sure about why the `Amount` struct was necessary. removed